modules/thermalmanager.dot

```diff
@@ -0,0 +1,161 @@
+/* -*- mode: c -*-
+ *
+ * Graph of thermal measurement control flow
+ *
+ * To create PNG image from this, use:
+ *
+ * % dot -T png thermalmanager.dot -o thermalmanager.png
```

Comment:
Nice-to-have: Would be cool if the packaging creates a `-doc` package with that `.png` already generated (for autodoc). Can build-depend on `graphviz` if possible, etc..